### ISSUE TITLE
can you explain your x_train,y_trian,x_test,y_test?

### ISSUE THREAD

**Issue by Aurora11111** [philipperemy/deep-speaker#45]
sir,can you explain your x_train,y_trian,x_test,y_test?it seems that the y_* isn't the label for data

**Comment by philipperemy** [comment]
Can you point where in the code @Aurora11111 ?

**Comment by Aurora11111** [comment]
@philipperemy 
in https://github.com/philipperemy/deep-speaker/blob/master/train_cli.py

Line 207

**Comment by philipperemy** [comment]
x_train, x_test are inputs to the models. From what I remember they contain MFCC features.

https://github.com/philipperemy/deep-speaker/blob/1e0b866d6290c62e6bd8f6c21bb545fc71a91768/utils.py#L19

y_train and y_test refer to the speaker id (one hot vector). So it's the label of the data.

**Comment by philipperemy** [comment]
During the softmax pre-training they are used.

HOWEVER, during the embeddings training we don't use the targets y to train the model.

**Comment by philipperemy** [comment]
Hope it's clear and makes sense for you!

**Comment by Aurora11111** [comment]
@philipperemy thank you,sir,I got it
